sdk/core/core-http/src/policies/generateClientRequestIdPolicy.ts

```diff
@@ -10,6 +10,29 @@ import {
   RequestPolicyOptions
 } from "./requestPolicy";
 
+/**
+ * Options for how HTTP connections generate the client request id header
+ * requests.
+ */
+export interface ClientRequestIdOptions {
+  /**
+   * The header name to use for the client request id
+   */
+  requestIdHeaderName?: string;
+  /**
+   * Whether or not to include a the generateClientRequestPolicyId pipeline
+   */
+  disable?: boolean;
+}
+
+/**
+ * Default options for ClientRequestId policy
```

Comment:
this should be `@internal` so it doesn't get picked up by the doc generator right?